src/views/catalog/templatescatalog/components/TemplatesCatalogDrawer/TemplatesCatalogDrawerCreateForm.tsx

```diff
@@ -74,6 +92,49 @@ export const TemplatesCatalogDrawerCreateForm: React.FC<TemplatesCatalogDrawerCr
         });
     };
 
+    const onCustomizeVM = () => {
+      setIsPreparingToCustomizeVM(true);
+      setCustomizeVMError(undefined);
+
+      // check for the required params and specify some default value for them to get the template processed successfully
+      // required parameters' values will be specified later  by the user anyway, during customization process
```

Comment:
You have an extra space between 'later' and 'by'.

Reply:
Oh, you are right! Will fixed that in some another PR! :)